cli.py

```
@@ -0,0 +1,344 @@
+                    now = time.time()
+                    if now - last_mine_time >= mine_interval:
+                        await self._try_mine_block()
+                        last_mine_time = now
+
+                # Keep event loop responsive
+                await asyncio.sleep(1)
+
+            except asyncio.CancelledError:
+                break
+
+    async def _try_mine_block(self):
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `_try_mine_block`

Add return type annotation: `None`

(ANN202)